src/io.cr

```diff
@@ -1123,14 +1123,14 @@ abstract class IO
   #
   # io2.to_s # => "hello"
   # ```
-  def self.copy(src, dst)
+  def self.copy(src, dst) : UInt64
```

Comment:
A little external argument maybe? I like to use them because they improve readabilty.
```suggestion
  def self.copy(src, to dst) : UInt64
```